tests/unit/cli/test_error_handler.py

```
@@ -141,6 +142,38 @@ def test_unexpected_error_json_format(self, capsys):
+        assert "🔍" in output
+        assert "\\u" not in output
+
+    def test_output_error_serializes_path_in_extra(self, capsys):
+        """_output_error must not crash on non-primitive extras like pathlib.Path."""
+        with pytest.raises(SystemExit) as exc_info:
+            _output_error(
+                "Bad path",
+                "PATH_ERROR",
+                json_output=True,
+                exit_code=1,
+                extra={"path": Path("/tmp/x")},
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/x"

(S108)